src/main/scala/nl/knaw/dans/easy/stage/Settings.scala

```diff
@@ -47,8 +47,6 @@ object Settings {
              urn: String,
              doi: String,
              otherAccessDoi: Boolean,
-             stageFileDataAsRedirectDatastreams: Boolean,
-             fileDataRedirectBaseUrl: Option[URL],
```

Comment:
Line 79 reads the disciplines from Fedora. At first sight it doesn't seem logical any more  ingest-flow still performs that task. So I'll guess I'll stick to the backward compatible call but it's a weird construction.